core/src/main/java/org/mule/runtime/core/internal/construct/FlowBackPressureException.java

```diff
@@ -21,7 +21,6 @@
  */
 public abstract class FlowBackPressureException extends MuleException {
 
-  static final String BACK_PRESSURE_ERROR_MESSAGE = "Flow '%s' is unable to accept new events at this time. Reason: %s";
```

Comment:
This change is unrelated, but I just removed it because it was unused. The message is actually taken from `CoreMessages` for i18n.